src/CanCommunity.ts

```diff
@@ -319,7 +319,7 @@ export class CanCommunity {
       actions: [
         {
           account: 'eosio.token',
-          authorization: [{ actor: input.creator, permission: 'active' }],
+          authorization: [{ actor: payer ? payer : input.creator, permission: 'active' }],
```

Comment:
We have a discussion in which you told me we need at least two signature.

* from creator
* from payer

Is the statement still valid? If so, is this change still valid?

Reply:
Let me represent the follow that payer want to create community for other one, that include two actions:

1. payer authorize to transfer CAT token to governance contract account to create community account, so `actor` here should be payer not creator, that's why I fix this. 
2. creator authorize to update community information with action `create` in governance contract

Reply:
@danielAlvess @manh-vv please help me resolve this MR. Looks like it has been here for a long time !?